scripts/create-dummy-docs-versions-in-library/README.md

```diff
@@ -198,11 +198,128 @@ catch
 [!INCLUDE [More about PnP PowerShell](../../docfx/includes/MORE-PNPPS.md)]
 ***
 
+# [CLI for Microsoft 365 with PowerShell](#tab/cli-m365-ps)
+
+```powershell
+# Usage example:
+#   .\Create-Bulk-Dummy-Documents.ps1 -WebUrl "https://contoso.sharepoint.com/sites/Intranet" -ListTitle "Documents" -ServerRelativeUrl "/sites/intranet/Shared Documents" -Type File -ItemsToCreate 5 -MajorVersions 6 -MinorVersionBeforeMajor 3 -FileToUse "D:\Temp\DummyFile.docx"
+#   .\Create-Bulk-Dummy-Documents.ps1 -WebUrl "https://contoso.sharepoint.com/sites/Intranet" -ListTitle "Documents" -ServerRelativeUrl "/sites/intranet/Shared Documents" -Type Folder -ItemsToCreate 5 -MajorVersions 6 -MinorVersionBeforeMajor 3 -FileToUse "D:\Temp\DummyFile.docx"
+[CmdletBinding()]
+param (
+  [Parameter(Mandatory = $true, HelpMessage = "Web url from which to create the files, e.g. https://contoso.sharepoint.com/sites/Intranet")]
+  [string]$WebUrl,
+  [Parameter(Mandatory = $true, HelpMessage = "Title of the list on which to create the documents or files")]
+  [string]$ListTitle,
+  [Parameter(Mandatory = $true, HelpMessage = "Server relative URL of the folder in which to create the documents")]
+  [string]$ServerRelativeUrl,
+  [Parameter(Mandatory = $true, HelpMessage = "Type of content to create, possible values are 'file' or 'folder'")]
+  [string]$Type,
+  [Parameter(Mandatory = $true, HelpMessage = "Amount of items to create")]
+  [int]$ItemsToCreate,
+  [Parameter(Mandatory = $true, HelpMessage = "Amount of major versions to create")]
+  [int]$MajorVersions,
+  [Parameter(Mandatory = $true, HelpMessage = "This will define the amount of minor versions that will be created before a major version is added")]
+  [int]$MinorVersionBeforeMajor,
+  [Parameter(Mandatory = $true, HelpMessage = "Path of the file to use when creating versions")]
+  [string]$FileToUse
+)
+begin {
+  $m365Status = m365 status 
+  if ($m365Status -match "Logged Out") {
+    m365 login
+  }
+  if ($Type.ToLower() -ne "file" -and $Type.ToLower() -ne "folder") {
```

Comment:
I dod not, I will change that.

Reply:
that's totally fine. It was only a suggestion as this would be an approach I would take but yours was fine as well. 
all in all whats most important is that I tested it and it works 👍👍.